parser.c

```diff
@@ -162,7 +162,11 @@ static bool parser_next_token(struct kanshi_parser *parser) {
 		} else if (ch == '"') {
 			parser->tok_type = KANSHI_TOKEN_STR;
 			parser->tok_str_len = 0;
-			return parser_read_quoted(parser);
+			return parser_read_quoted(parser, '"');
+		} else if (ch == '\'') {
+			parser->tok_type = KANSHI_TOKEN_STR;
+			parser->tok_str_len = 0;
+			return parser_read_quoted(parser, '\'');
```

Comment:
```suggestion
		if (ch == '"' || ch == '\'') {
			parser->tok_type = KANSHI_TOKEN_STR;
			parser->tok_str_len = 0;
			return parser_read_quoted(parser, ch);
```

Replace it with one conditional instead of two?

Reply:
Good catch :+1:

Reply:
My suggestion can't be applied as is, you need to remove the conditional above too.

Reply:
Aand... it fails the CI now, I guess applying this from GitHub web was a bad idea. I'll checkout locally and fix that.